base/show.jl

```diff
@@ -1422,28 +1445,36 @@ function print_matrix_repr(io, X::AbstractArray)
                 print(io, undef_ref_str)
             else
                 el = X[i,j]
-                compact ? showcompact_lim(io, el) : show(io, el)
+                show(io, el)
             end
         end
         if i < size(X,1)
-            println(io)
+            print(io, ";")
```

Comment:
The output looks like `[0.593082 0.91431; 0.275514 0.0228734]`. But looking at it again this code can be simplified to make that more obvious.